### PR TITLE
Disable node menu actions during map data load

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

**Review by SlimaneAmar — APPROVED** [review]

File: src/components/graph/menus/create-node-menu.js [hunk 1]
```diff
@@ -244,7 +246,7 @@ const CreateNodeMenu = ({
         PASTE_MODIFICATION_NODE: {
             onRoot: true,
             id: 'pasteNetworkModificationNode',
-            disabled: !isNodePastingAllowed(),
+            disabled: !isNodePastingAllowed() || mapDataLoading,
```
Comment (sBouzols) [inline src/components/graph/menus/create-node-menu.js:249]: why not using it in `isNodePastingAllowed` function directly ?

File: src/components/graph/menus/create-node-menu.js [hunk 2]
```diff
@@ -300,18 +302,22 @@ const CreateNodeMenu = ({
             onRoot: true,
             action: () => pasteSubtree(),
             id: 'pasteNetworkModificationSubtree',
-            disabled: !isSubtreePastingAllowed(),
+            disabled: !isSubtreePastingAllowed() || mapDataLoading,
```
Comment (sBouzols) [inline src/components/graph/menus/create-node-menu.js:305]: why not using it in `isSubtreePastingAllowed` function directly ?

File: src/components/graph/menus/create-node-menu.js [hunk 3]
```diff
@@ -300,18 +302,22 @@ const CreateNodeMenu = ({
...
         },
         REMOVE_SUBTREE: {
             onRoot: false,
             action: () => removeSubtree(),
             id: 'removeNetworkModificationSubtree',
-            disabled: isAnyNodeBuilding || !isSubtreeRemovingAllowed(),
+            disabled:
+                isAnyNodeBuilding ||
+                mapDataLoading ||
+                !isSubtreeRemovingAllowed(),
```
Comment (sBouzols) [inline src/components/graph/menus/create-node-menu.js:314]: same here

File: src/components/graph/menus/create-node-menu.js [hunk 4]
```diff
@@ -203,6 +204,7 @@ const CreateNodeMenu = ({
         CREATE_MODIFICATION_NODE: {
             onRoot: true,
             id: 'createNetworkModificationNode',
+            disabled: mapDataLoading,
```
Comment (sBouzols) [inline src/components/graph/menus/create-node-menu.js:207]: Why do we need to do this check ? Actualy it create an empty not built node somewhere, when does this create a mapDataLoading problem ?

Comment (etiennehomer) [reply src/components/graph/menus/create-node-menu.js:207]: Removed

**Review by sBouzols — APPROVED** [review]

**Pull request merged**